hbase-handler/pom.xml

```diff
@@ -255,6 +255,10 @@
           <groupId>commons-logging</groupId>
           <artifactId>commons-logging</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
can we drop all the exclusions from hadoop-common since it's declared in root pom.xml and simply use
````
 <dependency>
    <groupId>org.apache.hadoop</groupId>
    <artifactId>hadoop-common</artifactId>
    <classifier>tests</classifier>
    <scope>test</scope>
 <dependency>
````